modules/qt/nixos.nix

```
@@ -38,7 +41,7 @@ in
         lib.mkDefault "lxqt"
       else
         lib.mkDefault "qtct";
-    qt = {
+    qt = lib.mkDefault {
```

Due to https://github.com/danth/stylix/pull/388, we should not use `lib.mkDefault` to make the workaround slightly more convenient:

```suggestion
      qt = {
```